src/storage-client/src/client.rs

```diff
@@ -483,7 +503,7 @@ where
 
     fn new(parts: usize) -> PartitionedStorageState<T> {
         PartitionedStorageState {
-            parts: parts.try_into().expect("more than 4 billion partitions"),
```

Comment:
Changing this to `usize` requires doing `as` conversions below and add lints, do we need to change it?

Reply:
because `compute` uses `usize`, so I made them the same; I felt more comfortable changing storage code, ill add a todo to make them both u32's

Reply:
added a todo!